interface/billing/edit_payment.php

```diff
@@ -176,9 +176,10 @@
 				 }
 				else
 				 {
-				  sqlStatement("insert into ar_activity set "    .
+					 sqlStatement("insert into ar_activity set "    .
 					"pid = '"       . trim(formData("HiddenPId$CountRow"   )) .
```

Comment:
Please create a common callable function for inserting into ar_activity, and call that here and everywhere else you insert into it.